auto_py_to_exe/web/js/i18n.js

```diff
@@ -2291,6 +2359,10 @@ const supportedLanguages = [
     name: 'Italian (Italiano)',
     code: 'it',
   },
+  {
+    name: 'Slovak (Slovenčina)',
+    code: 'sk',
+  },
```

Comment:
You have checked " I have added the language to `supportedLanguages` in alphabetical order" but this is not. It should be under Serbian.